### DISCUSSION TITLE
DevConsole command

### DISCUSSION THREAD

**Discussion by TheHackerLxr** [kohls-admin/kohls-admin#68] [Ideas]
Opens roblox devconsole if you cant open it for some reason. Maybe if you are on mobile (useful to check logs on mobile in other games)

**Comment by kohltastrophe** [comment]
You can access this via the "/console" Roblox chat command, however we should offer a command for those who would like to use the admin.

**Reply by TheHackerLxr** [reply]
sometimes its disabled thats why I ask :(
